com.unity.render-pipelines.core/Runtime/Utilities/CoreUtils.cs

```diff
@@ -421,6 +421,24 @@ public static void SetRenderTarget(CommandBuffer cmd, RenderTargetIdentifier buf
             ClearRenderTarget(cmd, clearFlag, clearColor);
         }
 
+        /// <summary>
+        /// Set the current render texture.
+        /// </summary>
+        /// <param name="cmd">CommandBuffer used for rendering commands.</param>
+        /// <param name="colorBuffer">Color buffer RenderTargetIdentifier.</param>
+        /// <param name="colorLoadAction">Color buffer load action.</param>
+        /// <param name="colorStoreAction">Color buffer store action.</param>
+        /// <param name="depthLoadAction">Depth buffer load action.</param>
+        /// <param name="depthStoreAction">Depth buffer store action.</param>
+        /// <param name="clearFlag">If not set to ClearFlag.None, specifies how to clear the render target after setup.</param>
+        /// <param name="clearColor">If applicable, color with which to clear the render texture after setup.</param>
+        public static void SetRenderTarget(CommandBuffer cmd, RenderTargetIdentifier colorBuffer, RenderBufferLoadAction colorLoadAction, RenderBufferStoreAction colorStoreAction,
```

Comment:
this just exposes an existing CoreUtils binding to SRP, because the specific use case that we were missing was having the possibility of setting depth store actions without setting the depth attachment. Before this change any depth actions you would set in URP would be ignored by the engine because the API was just to passing those parameters down to the engine API.
In the only place where we are currently calling this only the clearColor parameter is needed. I can add a clear depth value too just for completeness? But wouldn't be used right now. WDYT?

Reply:
Why do you want to set depth store action when there is no depth attachment?
This only makes sense if colorBuffer is a RenderTargetIdentifier that refers to color and depth. But In that case it should have a different name.

Reply:
the depth attachment is used but being the backbuffer you cannot set it explicitly in URP, if you do it will crash because most devices handle this setup under the hood. You can look at the code calling this API to understand the limitation (SetRenderTarget in ScriptableRenderer.cs). The CoreUtils binding in C++ was already there, I just exposed it so it can be called in ScriptableRenderer.cs

So the only way for you to be able to change depth actions when dealing with the backbuffer is this (before my change depth actions were just ignored and never passed down to the engine)

Reply:
Then please don't call it `colorBuffer`.
https://docs.unity3d.com/ScriptReference/Rendering.CommandBuffer.SetRenderTarget.html calls it `rt` for this case and in CoreUtils it's `buffer`.

Reply:
makes sense, changed it to buffer